CMakeLists.txt

```diff
@@ -25,6 +25,12 @@ if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "armv7l")
 	set(CMAKE_C_FLAGS "-Ofast -fPIC -mfloat-abi=hard -mfpu=vfp -march=armv7-a -ffast-math -Wfloat-equal -Wshadow -Wcast-align -Wstrict-overflow=5 -Wwrite-strings -Waggregate-return -Wswitch-default -Wswitch-enum -Wformat=2 -g -Wall -Wno-sign-conversion -Wunreachable-code -Wstrict-prototypes")
 endif()
 
+if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "^mips")
+	set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE armhf)
```

Comment:
If(mips) set armhf?

I don't fully understand what is going on. But is that ok?

Also, 
mips generally refers to big endian. CI20 is mipsel (little endian)..